west.yml

```diff
@@ -182,4 +183,4 @@ manifest:
     path: nrf
     # This line configures west extensions which are currently only
     # for internal use by NCS maintainers.
-    west-commands: scripts/west-commands.yml
+    west-commands: scripts/west-commands.yml
```

Comment:
why did you / your editor decided to remove the newline here ?

This is causing compliance failure.

Reply:
fixed

Reply:
I'm apparently having issues with my editor, not on purpose.